objects/AllPlayerCards.15bb07/Unrelenting1.45386d.ttslua

```diff
@@ -1,5 +1,3 @@
-ASSEMBLY_POSITION = {69.08, 4, 36.45}
```

Comment:
Should there be more changes in this file?

Reply:
Nope, just a removal of a dead table (noticed this while looking at all the cards that can have chaos tokens on them)